### PR TITLE
Payload Support 1.1+

### PR DESCRIPTION
**Pull request by thehappydinoa**
Fix [Issue #248](https://github.com/hak5/bashbunny-payloads/issues/248)

### CODE REVIEW

**Comment by sundhaug92** [comment]
Seems like you're doing a lot: two payloads, an extension and an added variable. Looking at it: #271 appears fixed. Verifying and updating #248 based on this being merged.

**Comment by sundhaug92** [comment]
re #271: Seems like you forget to GET the variables needed by the payloads, so you're breaking them. Please push a commit that fixes that and then you'll have my stamp of approval.

**Comment by thehappydinoa** [comment]
@sundhaug92 Oops I totally forgot about the GETs.

File: payloads/library/credentials/BruteBunny/payload.txt [hunk 1]
```diff
@@ -28,17 +25,17 @@ sync;sleep 1;sync
...
 
 # wait for storage
-LED R G B 100
+LED STAGE1
 QUACK DELAY 6000
 QUACK GUI r
 QUACK DELAY 100
 # unleash the brute bunny
-LED B 100
+LED STAGE@
```
Comment (sundhaug92) [inline payloads/library/credentials/BruteBunny/payload.txt]: Typo

**Comment by thehappydinoa** [comment]
Fixed It

**Comment by sundhaug92** [comment]
HFR undo c1070e2

OK. Just for reference: 

* Fix compatibility with 1.1+ coding-standards
* Fix compatibility with languages other than the one used by the dev
* Adds "GET OS", which tries to detect the target OS
* Adds an extension for making Macs happy with the VID and PID
* Adds QUACK RUN LINUX as an alias for QUACK RUN UNITY

**Comment by thehappydinoa** [comment]
I am closing this PR as I have made a neater and more detailed PR #277

**Pull request closed without merging**